custom_components/zaptec/__init__.py

```diff
@@ -608,19 +617,28 @@ async def _trigger_poll(self, obj: ZaptecBase) -> None:
             _LOGGER.debug(
                 "Triggering poll %s of %s after %s seconds",
                 i,
-                obj.qual_id,
+                zaptec_obj.qual_id,
                 delta,
             )
             await self.async_refresh()
 
-    async def trigger_poll(self, obj: ZaptecBase) -> None:
+            # Trigger the poll for the children coordinators
+            for coord in children_coordinators:
+                await coord.trigger_poll()
```

Comment:
This is inside the for loop, so charger(s) will now trigger 4 times: 4, 9, 14 and 22 seconds after the triggering event. This is because the first round of polling is interrupted by a new trigger from the installation poller